doc/1/controllers/document/m-update/index.md

```diff
@@ -0,0 +1,68 @@
+---
+code: true
+type: page
+title: MUpdateAsync
+description: Updates documents.
+---
+
+# MUpdateAsync
+
+Updates multiple documents.
+
+Returns a partial error (error code 206) if one or more documents can not be updated.
```

Comment:
Never write `can not` (unless [in a few particular cases](https://grammarist.com/usage/cannot-or-can-not/)):

```suggestion
Returns a partial error (error code 206) if one or more documents cannot be updated.
```